PayloadPkg/OsLoader/OsLoader.c

```diff
@@ -1217,20 +1190,20 @@ BootOsImage (
     }
 
     //
-    // Init File System
+    // Parse A/B Slot Info
     //
     if (!EFI_ERROR (Status)) {
-      Status = InitBootFileSystem (OsBootOption, HwPartHandle, &FsHandle);
+      Status = ParseAbSlot (OsBootOption, HwPartHandle);
```

Comment:
Why we add A/B slot info here? If we need update A/B support, we could create a separate PR.
BTW, in the boot option, Image[2] is for A/B support, please check if you could use existing logic.

Reply:
Got it. This PR just moves A/B support out of InitBootFileSystem(). It is expected change so that it could work for all kind of image loading type.  Current SBL just provides the method to support OS A/B, it could be updated in actual use case late.